addons/website_event/models/event.py

```diff
@@ -73,16 +72,16 @@ def _set_show_menu(self):
     def _get_show_menu(self):
         self.show_menu = bool(self.menu_id)
 
-    def google_map_img(self, cr, uid, ids, zoom=8, width=298, height=298, context=None):
-        event = self.browse(cr, uid, ids[0], context=context)
-        if event.address_id:
-            return self.browse(cr, SUPERUSER_ID, ids[0], context=context).address_id.google_map_img()
+    def google_map_img(self, zoom=8, width=298, height=298):
```

Comment:
@tde-banana-odoo , I think we should remove google_map_img and google_map_link.

From template, we can call event_id.sudo().address_id.google_map_img and event_id.sudo().address_id.google_map_link.